modules/ROOT/pages/configure-tls-context-rtf-lb.adoc

```diff
@@ -1,73 +1,13 @@
-You can select keystores that are either PEM or non-PEM types. 
-* Optionally select a truststore to which to add the TLS context if you are using a truststore to store certificates trusted by the client. +
-A truststore is required if you are going to enable client authentication.
-* *Expiration Date* +
-Optionally select the expiration date for the certificate.
-. Optionally xref:enable-client-authentication.adoc[enable client authentication]. 
-. Optionally xref:cipher-suites.adoc[select ciphers]. 
-. Click *Save*.
-
-[[import_tls_rtf]]
-== Import a TLS Context to Runtime Fabric
-
-. In *Runtime Manager > Runtime Fabrics > Inbound Traffic*, select one of the following options: +
-* *Upload PEM* +
-For information about this option, see xref:runtime-fabric::enable-inbound-traffic.adoc[Enable Inbound Traffic on Anypoint Runtime Fabric].
-* *Upload JKS* +
-For information about this option, see xref:runtime-fabric::enable-inbound-traffic.adoc[Enable Inbound Traffic on Anypoint Runtime Fabric].
-* *Import from Secrets Manager*. +
-This option is automatically selected for instances of Runtime Fabric for which inbound traffic was enabled before the PEM and JKS upload options became available. +
-This option imports a TLS context from the secrets manager, and supports advanced configuration such as creating a TLS context, mutual authentication, selecting ciphers, and selecting TLS versions.
-. Select the environment the secrets group that contains the TLS context is associated with.
-. Select the secrets group where the TLS context is configured.
-. Select the TLS context to use for Runtime Fabric. +
-You can click *View Details* to see details of the TLS context you are importing. 
-
-[[tls_support_rtf]]
-== TLS Support on Anypoint Runtime Fabric
-
-OpenSSL 1.1.1 and TLS 1.3 provide the following security enhancements and performance improvements over TLS 1.2:
-
-* 2x or greater TLS 1.2 connection performance throughput boost over an earlier Runtime Fabric internal load balancer running on OpenSSL 1.0.2
-* Reduction of one round trip in a full handshake for TLS 1.3 vs. TLS 1.2
-* Added TLS 1.2 and TLS 1.3 ChaCha20-Poly1305 ciphers for better mobile and IoT device support
-* TLS 1.3 protection against downgrade attacks
-* Continued support for both RSA and ECDSA keys
-* Removal of obsolete and insecure features from TLS 1.2 and TLS 1.1. +
-See xref:release-notes::anypoint-security/anypoint-security-rtf-edge.adoc[Anypoint Security Edge Release Notes] for a list of supported and deprecated ciphers.
-
-The TLS 1.3 protocol is enabled by default for those clients that support it.
+If your environment requires advanced configuration such as client authentication (mutual authentication), selecting ciphers, and selecting TLS versions, you can import a TLS context from the secrets manager. Refer to xref:runtime-fabric::configure-adv-tls-context.adoc[Import a TLS Context from Secrets Manager (Advanced)].
```

Comment:
Would it be retain meaning to make the verbs parallel?
```suggestion
If your environment requires advanced configuration such as authenticating clients (mutual authentication), selecting ciphers, and selecting TLS versions, you can import a TLS context from the secrets manager. Refer to xref:runtime-fabric::configure-adv-tls-context.adoc[Import a TLS Context from Secrets Manager (Advanced)].
```
Also, I can't tell if this is a link? If it is, please go ahead and refer to the title but do not make it live; move the live link to See Also (which we might rename to Related Information to avoid using "see" re. accessibility).

Reply:
Leaving as is - "client authentication" is what's in the UI, and we want to relate that to "mutual authentication". Engineering asked to have the link inline since it's kind of a special case.